language/tools/move-unit-test/src/test_runner.rs

```diff
@@ -396,6 +400,8 @@ impl SharedTestingConfig {
             .unwrap_or_else(|e| panic!("Unable to build stackless bytecode: {}", e));
 
             if model.has_errors() {
+                let mut stderr = StandardStream::stderr(ColorChoice::Always);
+                model.report_diag(&mut stderr, Severity::Error);
```

Comment:
Why the change here?

Reply:
Debugging errors was hard. This just prints more concrete info.